.prettierrc.js

```diff
@@ -1,7 +1,4 @@
 module.exports = {
-    jsxBracketSameLine: true,
     trailingComma: 'all',
-    tabWidth: 4,
```

Comment:
This change might be controversial. Prettier's default is 2 spaces.

Reply:
I think it is pretty common in JS circles.